ui/tests/acceptance/secrets/backend/pki/role-test.js

```diff
@@ -14,25 +14,20 @@ module('Acceptance | secrets/pki/create', function(hooks) {
     return authPage.login();
   });
 
-  const mountAndNav = async () => {
+  test('it creates a role and redirects', async function(assert) {
+    // UPGRADE TODO: Getting error:
+    // Promise rejected before "it creates a role and redirects meep": Assertion Failed: You cannot use the same root element (#ember-testing) multiple times in an Ember.Application
     const path = `pki-${new Date().getTime()}`;
     await enablePage.enable('pki', path);
     await settled();
     await editPage.visitRoot({ backend: path });
     await settled();
-    return path;
-  };
-
-  test('it creates a role and redirects', async function(assert) {
-    // UPGRADE TODO: Getting error:
-    // Promise rejected before "it creates a role and redirects meep": Assertion Failed: You cannot use the same root element (#ember-testing) multiple times in an Ember.Application
-    const path = await mountAndNav(assert);
     await editPage.createRole('role', 'example.com');
     await settled();
     assert.equal(currentRouteName(), 'vault.cluster.secrets.backend.show', 'redirects to the show page');
-    assert.ok(showPage.editIsPresent, 'shows the edit button');
-    assert.ok(showPage.generateCertIsPresent, 'shows the generate button');
-    assert.ok(showPage.signCertIsPresent, 'shows the sign button');
+    assert.dom('[data-test-edit-link]').exists('shows the edit button');
+    assert.dom('[data-test-credentials-link]').exists('shows the generate button');
+    assert.dom('[data-test-sign-link]').exists('shows the sign button');
```

Comment:
I remember implementing some of this on the Ember upgrade, and yeah, I think it's the right approach.